### ISSUE TITLE
A10y: Modals should have focussed when opened

### ISSUE THREAD

**Issue by rugk** [PrivateBin/PrivateBin#1801]
## Steps to reproduce
> the focus is not set directly upon the boxes but one would need to press tab once. After closing of dialogs the focus should be on the invoking button,

from https://github.com/PrivateBin/PrivateBin/issues/1794#issuecomment-3968486674

## Additional information

/cc @cloudscape-germany
